wolfcrypt/src/rsa.c

```diff
@@ -2356,8 +2356,19 @@ static int wc_RsaFunctionSync(const byte* in, word32 inLen, byte* out,
 
 #if defined(WOLFSSL_SP_MATH)
     (void)rng;
+    #ifndef WOLFSSL_HAVE_SP_RSA
+    (void)in;
+    (void)inLen;
+    (void)out;
+    (void)outLen;
+    (void)type;
+    (void)key;
+    #error RSA SP option invalid (enable WOLFSSL_HAVE_SP_RSA or disable WOLFSSL_SP_MATH)
+    return NOT_COMPILED_IN;
+    #else
```

Comment:
Seems kind of odd....you're avoiding lots of warnings with the `(void)` stuff but then have an `#error` which makes the `(void)` stuff ineffective.  Perhaps you are trying to avoid a lot of unused variable warnings so that if the error is hit, the person compiling won't be distracted by them?

Reply:
Correct, the unused variables are just distracting. The real error is the invalid configuration. I did consider adding yet another macro to suppress the error, but I don't think this is a compile case folks should be using.

Reply:
Sounds good to me. Just made me do a double take. :)